wscript

```diff
@@ -290,6 +290,10 @@ iconv support use --disable-iconv.",
         'desc' : 'Lua',
         'func': check_lua,
     }, {
+        'name' : '--javascript',
+        'desc' : 'Javascript (MuJS backend)',
+        'func':  check_statement('mujs.h', 'js_setreport(js_newstate(0, 0, 0), 0)', lib='mujs'),
+    }, {
```

Comment:
I checked with the MuJS guys - they currently don't. If they add it at some point, I'll update `wscript` to use it.